kolibri/plugins/bloompub_viewer/assets/src/views/BloomPubRendererIndex.vue

```diff
@@ -162,58 +156,21 @@
         this.loading = false;
         this.$emit('error', err);
       });
-      let storageUrl = this.defaultFile.storage_url;
-      if (!this.isBloom) {
```

Comment:
all of this code deletion is because in the bloompub renderer index, we wouldn't be rendering non-bloom files? or is it something else

Reply:
Just that, yes.